examples.patch

```diff
@@ -1,102 +1,103 @@
-From d709f031964a1c5459e79ee4151da35f106514ec Mon Sep 17 00:00:00 2001
+From 651e3fdff67c7760f5321cd10d7786665d696d99 Mon Sep 17 00:00:00 2001
```

Comment:
This is an upstream bugfix/improvement patch.  It should be submitted as a PR upstream so that we don't have to continue to carry it.

Reply:
it has been removed